src/nodes/delta/delta.jl

```diff
@@ -0,0 +1,245 @@
+export DeltaFn
+
+struct DeltaFnCallableWrapper{F} end
+
+(::Type{DeltaFnCallableWrapper{F}})(args...) where {F} = F.instance(args...)
+
+struct DeltaFn{F} end
+struct DeltaFnNode{F, N, L, M} <: AbstractFactorNode
+    fn::F
+
+    out::NodeInterface
+    ins::NTuple{N, IndexedNodeInterface}
+
+    localmarginals :: L
+    metadata       :: M
+end
+
+as_node_symbol(::Type{DeltaFn{ReactiveMP.DeltaFnCallableWrapper{F}}}) where {F} = Symbol(:DeltaFn, string(F))
+
+functionalform(factornode::DeltaFnNode{F}) where {F}      = DeltaFn{DeltaFnCallableWrapper{F}}
+sdtype(factornode::DeltaFnNode)                           = Deterministic()
+interfaces(factornode::DeltaFnNode)                       = (factornode.out, factornode.ins...)
+factorisation(factornode::DeltaFnNode{F, N}) where {F, N} = ntuple(identity, N + 1)
```

Comment:
In American spelling it would be `factorization`, not sure what the consensus is here.